pages/cart/cart.html

```diff
@@ -0,0 +1,318 @@
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- MAIN SECTION -->
+    <main class="main-section">
+      <!-- CART PRODUCTS -->
+      <section class="cart-products">
+        <h2 class="cart-title">
+          My Cart <small class="cart-total-items">(4 Items)</small>
+        </h2>
+
+        <section class="cart-container">
+          <section class="cart-items">
+            <div class="horizontal-card">
+              <div class="horizontal-card-img">
+                <img src="../../images/product-1.jpg" alt="product-1-img" />
+              </div>
+              <div class="horizontal-card-content">
+                <div class="horizontal-card-text">
+                  <h2 class="horizontal-card-title">Red T-Shirt</h2>
+                  <label for="quantity" class="quantity-label">Quantity</label>
+                  <input
+                    type="number"
+                    id="quantity"
+                    class="quantity"
+                    value="1"
+                  />
+                  <p class="size">Size: <strong>XL</strong></p>
+                  <div class="card-rating">
+                    <ul class="stars">
+                      <li class="star star-filled">
+                        <i class="fas fa-star"></i>
+                      </li>
+                      <li class="star star-filled">
+                        <i class="fas fa-star"></i>
+                      </li>
+                      <li class="star star-filled">
+                        <i class="fas fa-star"></i>
+                      </li>
+                      <li class="star star-filled">
+                        <i class="fas fa-star"></i>
+                      </li>
+                      <li class="star"><i class="fas fa-star"></i></li>
+                    </ul>
+                  </div>
+                  <p class="card-price">
+                    ₹499 <small class="price-before">₹1299</small
+                    ><small class="price-discount">62%OFF</small>
+                  </p>
+                </div>
+                <div class="card-action">
+                  <button class="cart-btn btn-outline-primary">
+                    Move to Wishlist
+                  </button>
+                  <button class="cart-btn btn-outline-danger">Remove</button>
+                </div>
+              </div>
+            </div>
+            <div class="horizontal-card">
+              <div class="horizontal-card-img">
+                <img src="../../images/product-1.jpg" alt="product-1-img" />
+              </div>
+              <div class="horizontal-card-content">
+                <div class="horizontal-card-text">
+                  <h2 class="horizontal-card-title">Red T-Shirt</h2>
+                  <label for="quantity" class="quantity-label">Quantity</label>
+                  <input
+                    type="number"
+                    id="quantity"
+                    class="quantity"
+                    value="1"
```

Comment:
The label `for` and input's `id` attribute are same everywhere. I would suggest to change because if the user clicks on any label only the first `input` gets targeted